api-v3-beta.rst

```diff
@@ -0,0 +1,53 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
```

Comment:
We should show off some of the queries here. Like, "get a list of my projects filtered by X" that will work for anyone logged in.

Reply:
I tried to archive this goal in the small list of new features. But it seems that it didn't work well.

I think I prefer to improve that list and make it work than add the feature in this paragraph. I'll try that.